conceptual/Npgsql/connection-string-parameters.md

```diff
@@ -84,7 +84,7 @@ Host Recheck Seconds      | Controls for how long the host's cached state will b
 Parameter                | Description                                                                                          | Default
 ------------------------ | ---------------------------------------------------------------------------------------------------- | ----------
 Options<sup>[1](#misc_sup_1)</sup> | Specifies any valid [PostgreSQL connection options](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-OPTIONS) (e.g. `Options=-c synchronous_commit=local`). Introduced in 5.0. | PGOPTIONS
-Application Name         | The optional application name parameter to be sent to the backend during connection initiation.      |
+Application Name<sup>[2](#misc_sup_2)</sup> | The optional application name parameter to be sent to the backend during connection initiation.      | PGAPPNAME
```

Comment:
Can you please align this to the other entries (e.g. see `Client Encoding` bellow, where PGCLIENTENCODING is simply in the Default column? We don't provide yaml examples for Kubernetes for any other option (and that seems out of scope for these docs - why specifically Kubernetes and not other systems, etc.).

Reply:
This is updated accordingly, thanks for the feedback!